docs/source/libraries/lightgbm.rst

```diff
@@ -51,6 +50,14 @@ for ``lightgbm.LGBMClassifer`` and ``lightgbm.LGBMRegressor``:
   estimator. Set it to True if you're passing ``vec``,
   but ``doc`` is already vectorized.
 
+``lightgbm.Booster`` estimator accepts one more optional argument:
+
+* ``is_regression`` - True if solving a regression problem
+  ("objective" starts with "reg")
+  and False for a classification problem.
+  If not set, regression is assumed for a single target estimator
+  and proba will not be shown unless the ``target_names`` is defined as a list with length of two.
```

Comment:
This seems to contradict a note above (line 25, "target_names is ignored") - does the note about target_names apply only to classifier/regressor, but not for Booster?

Reply:
for a single target estimater, booster cannot recognize wheather it is a regression problem or not. We assume it is regression in default, unless users set it as  a classification problem by assigning  'target names' input [0,1] etc. Only in this case  'target names' is used.  Should I remove " target names is ignored" to prevent confusion？

Reply:
target names is still ignored for classifier/regressor, right? I think it makes sense to clarify it - just say that it is ignored for LGBMClassifier / LGBMRegressor, but used for lightgbm.Booster.

"is defined as a list with length of two" - sohuld it be 2 elements, or 2+ is also supported?

Reply:
Well, for single target booster,  only 2 elements target_name point booster to classification. Otherwise, this booster is regression (there is risk user assign wrong number of elements like 3, but not sure how eli5 will raise error. 2+ should not support here).